Automattic/wp-calypso
# FSE: Add block pattern infrastructure

#### Changes proposed in this Pull Request

* Adds an FSE module to distribute block patterns.
* Adds three block patterns as examples.

#### Testing instructions

* Make sure to have the latest version of Gutenberg built.
* Apply this PR.
* Load the Editor and switch to "Block Patterns" in the toolbar.
* Insert a block pattern.

### TODO
- [ ] Decide where to host images used in block patterns. headstartdata? As part of the plugin? Use [Core's images](https://core.trac.wordpress.org/ticket/49580)?
- [ ] Decide whether we need to support json files, given that we probably want to always localize pattern titles?
- [x] See how to best distribute translations. Not sure of the `full-site-editing` text domain even works?
